docs/custom_domains.rst

```diff
@@ -112,6 +81,68 @@ for our own domains including ``*.readthedocs.io``, ``*.readthedocs-hosted.com``
 .. _Strict Transport Security header: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Strict-Transport-Security
 
 
+Troubleshooting
++++++++++++++++
+
+SSL certificate issue delays
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The status of your domain validation and certificate can always be seen on the details page for your domain
+under :guilabel:`Admin` > :guilabel:`Domains` > :guilabel:`YOURDOMAIN.TLD (details)`.
+
+* For |org_brand|, domains are usually validated and a certificate issued within minutes.
+  However, if you setup the domain in Read the Docs without provisioning the necessary DNS changes
+  and then update DNS hours or days later,
+  this can cause a delay in validating because there is an exponential back-off in validation.
+  Loading the domain details in the Read the Docs dashboard and saving the domain again will force a revalidation.
+* For |com_brand|, domains can take up to a couple days to validate and issue a certificate.
+  To avoid any downtime in moving a domain from somewhere else to Read the Docs,
+  it is possible to validate the domain and provision the certificate before pointing your domain to Read the Docs.
+
+Certificate authority authorization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Certificate authority authorization (CAA) is a security feature that allows domain owners to limit
+which certificate authorities (CAs) can issue certificates for a domain.
+This is done by setting CAA DNS records for your domain.
+CAA records are typically on the root domain, not subdomains
+since you can't have a CNAME and CAA record for the same subdomain.
+Here's an example for palletsprojects.com:
+
+.. prompt:: bash $, auto
+
+    $ dig CAA +short palletsprojects.com
+    0 issue "digicert.com"
+    0 issue "comodoca.com"
+    0 issue "letsencrypt.org"
+
+If there are CAA records for your domain that do not allow the certificate authorities that Read the Docs uses,
+you may see an error message like ``pending_validation: caa_error: YOURDOMAIN.TLD``
+in the Read the Docs dashboard for your domain.
+You will need to update your CAA records to allow us to issue the certificate.
+
+* For |org_brand|, we use Cloudflare which uses Digicert as a CA. See the `Cloudflare CAA FAQ`_ for details.
+* For |com_brand|, we use AWS Certificate Manager as a CA. See the `Amazon CAA guide`_ for details.
```

Comment:
I do think an example of what a "proper" record looks like for confirmation would be useful, but maybe a bit much.

Reply:
There's a community example a few lines up (palletsprojects.com). Since this is outside the tab, it's not as nice to provide an example .com and a .org example.